prime-router/docs/design/features/0006-hl7-field-truncation.md

```diff
@@ -0,0 +1,55 @@
+# HL7 Field Truncation
```

Comment:
Why is this here? All changes in this PR should relate only to #10545. Same comment for the changes in .github folder. I think this branch is out of sync with master? I will ignore the other cases where weird things show up but should fix this before merging.

Reply:
I am not sure why it is here and how.